.idea/workspace.xml

```diff
@@ -177,6 +149,13 @@
       <workItem from="1524566730453" duration="65000" />
       <workItem from="1524566842130" duration="22000" />
       <workItem from="1524566931346" duration="689000" />
+      <workItem from="1524636067788" duration="310000" />
```

Comment:
are these numbers auto generated?  Else how are these numbers obtained/decided?

Reply:
@rpfondeker This file is used maintained by Ruby Mine, and I don't think they affect the operation of the gem